src/v2/Apps/Partner/Components/Overview/ShowBannersRail.tsx

```diff
@@ -0,0 +1,220 @@
+import React, { useState } from "react"
+import {
+  Box,
+  BoxProps,
+  Carousel,
+  CarouselCell,
+  CarouselRail,
+  ProgressDots,
+  Swiper,
+  SwiperCell,
+  SwiperRail,
+} from "@artsy/palette"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { ShowBannersRail_partner } from "v2/__generated__/ShowBannersRail_partner.graphql"
+import { ShowBannersRailRendererQuery } from "v2/__generated__/ShowBannersRailRendererQuery.graphql"
+import { compact, take, uniqBy } from "lodash"
+import { ShowBannerFragmentContainer } from "../PartnerShows"
+import { useSystemContext } from "v2/Artsy"
+import { ShowBannersRailPlaceholder } from "./ShowBannersRailPlaceholder"
+import { Media } from "v2/Utils/Responsive"
+
+interface ShowBannersRailProps extends BoxProps {
+  partner: ShowBannersRail_partner
+}
+
+export const ShowBannersRailContainer: React.FC<{
+  children: JSX.Element[]
+  onChange?(index: number): void
+}> = ({ children, ...rest }) => {
+  return (
+    <>
+      <Media greaterThan="xs">
+        <Carousel
+          {...rest}
+          Cell={React.forwardRef((props, ref) => {
+            return (
+              <CarouselCell
+                {...props}
+                ref={ref as any}
+                display="inline-flex"
+                width="100%"
+                pr={0}
+              />
+            )
+          })}
+          Rail={props => {
+            return <CarouselRail {...props} transition="none" display="block" />
+          }}
+        >
+          {children}
+        </Carousel>
+      </Media>
+      <Media at="xs">
+        <Swiper
```

Comment:
It's not clear to me if it'd be any better UX-wise but if you want scroll-snapping `Swiper` accepts a `snap` prop